cvat/apps/api_tokens/serializers.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import textwrap
+from typing import Any
+
+from rest_framework import serializers
+
+from cvat.apps.engine.serializers import BasicUserSerializer
+
+from . import models
+
+
+class ApiTokenReadSerializer(serializers.ModelSerializer):
+    owner = BasicUserSerializer(required=False)
+
+    value = serializers.CharField(
+        source="raw_token",
```

Comment:
Why not rename this to `value` too? Seems like an unnecessary discrepancy.

Reply:
I think `raw_token` is a much more clear name than `value`, from the backed developer perspective.